### PR TITLE
Factory: self serve workflow documentation

### PR DESCRIPTION
**Pull request by EmbeddedAndroid**
It's happening!

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 2c981acf07b947f4693d962ce2298c1c6cf471c9 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/336/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 19b27a0acddbbd36345377d64cd32bf6f54f52d1 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/337/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for dc8bbab437a71bcdf1d0cb5546e664c5dc214645 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/339/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 64663db8be82a66f1358c331ff2647845ddf48ff are browesable at: https://ci.foundries.io/projects/fio-docs/builds/340/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Here's the actual rendering: https://ci.foundries.io/projects/fio-docs/builds/340/docs/artifacts/html/customer-factory/getting-started.html

File: source/customer-factory/getting-started.rst [hunk 1]
```diff
@@ -5,10 +5,128 @@ Getting Started
...
 To get started using the FoundriesFactory please follow the steps below:
 
-#. Login to https://app.foundries.io and generate an access token.
+1. Create an account at https://app.foundries.io if you do not already have one.
+
+  You can create an account with our system or use your existing Github or Google account.
+
+2. Verify your email address
+
+  When you create an account, an automated email is sent to you to verify your email. Following the link in the email to verify your account.
```
Comment (ricardosalveti) [inline source/customer-factory/getting-started.rst]: s/following/follow/g?

File: source/customer-factory/getting-started.rst [hunk 2]
```diff
@@ -5,10 +5,128 @@ Getting Started
...
+      :alt: Factory Billing Account
+      :align: center
+      :width: 3in
+
+   Add a **Billing Account**
+
+   .. figure:: /_static/factory-add-account.png
+      :alt: Factory Account
+      :align: center
+      :width: 3in
```
Comment (doanac) [inline source/customer-factory/getting-started.rst]: I'm not sure we need a picture of the actual "Add billing account" button.

File: source/customer-factory/getting-started.rst [hunk 3]
```diff
@@ -5,10 +5,128 @@ Getting Started
 
 To get started using the FoundriesFactory please follow the steps below:
 
-#. Login to https://app.foundries.io and generate an access token.
+1. Create an account at https://app.foundries.io if you do not already have one.
```
Comment (ricardosalveti) [inline source/customer-factory/getting-started.rst]: Do we need to manually list the numbers per entry here? Might be harder to maintain but fine if there is no easy way with the rst formatting.

File: source/customer-factory/getting-started.rst [hunk 4]
```diff
@@ -5,10 +5,128 @@ Getting Started
...
+      :width: 3in
+
+   Choose the **Personal** or **Enterprise** subscription.
+
+   .. figure:: /_static/factory-subscription.png
+      :alt: Factory Subscription
+      :align: center
+      :width: 3in
+
+   Refer to https://foundries.io/pricing/ for details about each subscription.
```
Comment (doanac) [inline source/customer-factory/getting-started.rst]: Same thing here. I'm not sure we need a picture of the "Add subscription" button.

File: source/customer-factory/getting-started.rst (hunk 1 above)
Comment (EmbeddedAndroid) [reply source/customer-factory/getting-started.rst]: Fixed.

File: source/customer-factory/getting-started.rst (hunk 3 above)
Comment (EmbeddedAndroid) [reply source/customer-factory/getting-started.rst]: I tried to use the #. notation but it gave each element 1. I'm sure I have a formatting issue somewhere.

File: source/customer-factory/getting-started.rst (hunk 2 above)
Comment (EmbeddedAndroid) [reply source/customer-factory/getting-started.rst]: Will remove.

File: source/customer-factory/getting-started.rst (hunk 4 above)
Comment (EmbeddedAndroid) [reply source/customer-factory/getting-started.rst]: Ditto.

**Comment by doanac** [comment]
Docs for cf33e247ccea681f20615225e5820fd515addbcd are browesable at: https://ci.foundries.io/projects/fio-docs/builds/341/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for a8d3d7a2eb9fcc8fa4da6e18445cdbf5dd8f7cdd are browesable at: https://ci.foundries.io/projects/fio-docs/builds/342/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for e8d5229b501c09300811b5bd664039bcdd45d4b3 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/343/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 0ce9048359bee4f103e3e5db79d395eb7532b477 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/344/docs/artifacts/html/index.html

File: source/customer-factory/getting-started.rst (hunk 3 above)
Comment (EmbeddedAndroid) [reply source/customer-factory/getting-started.rst]: I moved to using subtitles, now the headings are in bold and has a ToC.

**Comment by EmbeddedAndroid** [comment]
@doanac @rsalveti @mike-scott have a look now, should have addressed your feedback.

**Pull request merged**